src/test/java/picard/sam/testers/SamFileTester.java

```diff
@@ -343,19 +344,23 @@ public void addMatePair(final String readName,
                 isDuplicate1, isDuplicate2, cigar1, cigar2, strand1, strand2, firstOnly, record1NonPrimary, record2NonPrimary, defaultQuality);
     }
 
-    protected abstract void test();
+    protected abstract void test() throws IOException;
 
     /**
      * Sets up the basic command line arguments for input and output and runs instanceMain.
      */
     public void runTest() {
-        final File input = createInputFile();
-
-        output = new File(outputDir, "output.sam");
-        args.add("INPUT=" + input.getAbsoluteFile());
-        args.add("OUTPUT=" + output.getAbsoluteFile());
-        Assert.assertEquals(runPicardCommandLine(args), 0);
-        test();
+        try {
+            final File input = createInputFile();
+
+            output = new File(outputDir, "output.sam");
+            args.add("INPUT=" + input.getAbsoluteFile());
+            args.add("OUTPUT=" + output.getAbsoluteFile());
+            Assert.assertEquals(runPicardCommandLine(args), 0);
+            test();
+        } catch (IOException ex) {
+            ex.printStackTrace();
```

Comment:
Is this really what's wanted here? Under what conditions do we want a test to ignore a thrown error?